twindle-cli/renderer/pdf/templates/Thread.hbs

```diff
@@ -215,8 +216,7 @@
         </svg>
         {{/if}}
         <span class="user-handle">
-          @
-          {{common.user.username}}
+          @{{common.user.username}}
```

Comment:
The problem here is that when you format your document, its going to put them on separate lines again, and we'll have the same problems again.

Using handlebars helpers can help with that . See https://stackoverflow.com/questions/29666047/sane-way-to-concat-string-and-variable-in-handlebars-js-helper-argument

Reply:
Can you explain a bit more what you are trying to say ?

Reply:
What I'm saying is that when you're using something like VSCode with Prettier as a formatter, and you hit your shortcut keys to format your code, prettier beautifies the code, and it sometimes break lines off. That is what is gonna happen. It's going to  put the {{common.user.username}} on a separate line again, and we'll have the space again between the @ and username.

That's why we need to concatenate these 2, like '@' + common.user.username. The SO link I sent you has the code to deal with the concatenation